jekyll/_cci2/workflows.md

```diff
@@ -216,6 +216,71 @@ workflows:
 
 In the example, `filters` is set with the `branches` key and the `only` key with the branch name. Any branches that match the value of `only` will run the job. Branches matching the value of `ignore` will not run the job. See the [Sample Sequential Workflow config with Branching](https://github.com/CircleCI-Public/circleci-demo-workflows/blob/sequential-branch-filter/.circleci/config.yml) for a full example.
 
+## Git Tag Job Execution
+Git tags must be declared in each job and in the workflow. CircleCI will not build Git tags unless they are specified under each job key. The following example shows a workflow with jobs filtered to ignore tags starting with `testing-`. 
+
+The following `build` job example will run for all branches, and all tags, except those starting with “testing-”.
+
+```
+workflows:
+  version: 2
+  build-workflow:
+    jobs:
+      - build:
+          filters:
+            tags:
+              ignore: /testing-.*/
+```
+
+The following example runs the `build` job for all branches, and all tags and runs the `deploy` job for **no** branches, and all tags starting with `config-test`. **Note:** the `build` job must have a `filters` `tags` key or it will **not** build for any tags.
```

Comment:
"the `build` job must have a `filters` `tags` key or it will **not** build for any tags." ->

"the `build` job must have a `filters` `tags` key or it will **not** build for any tag.  In which case, the `deploy` job will also not run because one of its `build` dependency is not satisfiable."